libsplinter/src/error/constraint_violation.rs

```diff
@@ -164,17 +165,17 @@ impl fmt::Display for ConstraintViolationError {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
         let mut format_string = String::new();
         if let Some(store) = &self.store {
-            format_string += &format!("{} ", store);
+            write!(format_string, "{}", store)?;
         }
 
         if let Some(operation) = &self.operation {
-            format_string += &format!("{}: ", operation);
+            write!(format_string, "{}", operation)?;
```

Comment:
This is missing a space and a `:`, causing a test failure. 
This results in 
`TestStoretest_opUnique constraint violated` 
where it should be
 `TestStore test_op: Unique constraint violated`

Reply:
Added the spaces and :s back in